man/journal-upload.conf.xml

```diff
@@ -129,6 +129,29 @@
         <listitem><para>Takes a boolean value, enforces using compression without content encoding negotiation.
         Defaults to <literal>false</literal>.</para>
 
+        <xi:include href="version-info.xml" xpointer="v258"/></listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Header=</varname></term>
+
+        <listitem><para>HTTP header which should be added to each request to URL.</para>
+
+        <para>Header name can contains alphanumeric values, "_" and "-" symbols additionally.</para>
+
+        <para>This option may be specified more than once, in which case all listed headers will be set. If
+        the same header name is listed more than once, all its unique values will be concatenated with comma.
+        </para>
+
+        <para>Example:
+        <programlisting>
+          Header=HeaderName: HeaderValue
+          Header=HeaderName: NewValue
+          Header=HeaderName: HeaderValue
+        </programlisting>
```

Comment:
hmm, some renderings of docbook really care about leading whitespace, hence please drop any leading whitespace within `<programlisting>` tags. and ideally even place opening tag on same line as first contents line without separating whitespace, and the closing line after it without separating whitespace